pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v1/Namespaces.java

```diff
@@ -187,7 +208,17 @@ public void createNamespace(@PathParam("property") String property, @PathParam("
             policies.bundles = getBundles(defaultNumberOfBundles);
         }
 
-        internalCreateNamespace(policies);
+        internalCreateNamespace(policies)
```

Comment:
line 196 `validateClusterForTenant(namespaceName.getTenant(), namespaceName.getCluster())` looks like a sync method that can be async.